metrics-service-ui/frontend/src/components/Dashboard/Dashboard.jsx

```diff
@@ -9,11 +9,56 @@
  */
 
 import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import React, { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+
+const useStyles = makeStyles((theme) => ({
+    formControl: {
+        margin: theme.spacing(1),
+        minWidth: 120,
+    },
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+}));
 
 export default function Dashboard() {
+    const classes = useStyles();
+    const [currentStream, setCurrentStream] = useState('DISCOVERABLECLIENT');
```

Comment:
Any reason to specifically have DISCOVERABLECLIENT here?

Reply:
It's because it's the default value?

Reply:
Yes it's the default value for now, fixing this is tracked in #1854 and I actually have an unmerged solution on apiml/GH1854/fix_hardcoded_metrix_ui